### DISCUSSION TITLE
URL과 URI의 차이점이 무엇일까요?

### DISCUSSION THREAD

**Discussion by Jay13Jeong** [Jay13Jeong/soo-je-webserver#9] [Q&A] [answered]
URL과 URI의 차이점이 무엇일까요?

**Comment by FeFe2200** [comment, marked as answer]
예시로 "http://www.example.com/index.html?page=1232950&id=776"이면, 

URL은 http에서 사용되는 주소, 도메인주소와 파일의 경로까지를 지칭합니다. "http://www.example.com/index.html"이고,

URI는 URL뒤에 붙는 자원의 식별자까지를 포함한  "http://www.example.com/index.html?page=1232950&id=776" 부분을 지칭합니다.
